### ISSUE TITLE
Focus on one type of POI

### ISSUE THREAD

**Issue by rouelibre1** [Zverik/every_door#406]
My use case is for bicycle parking, but it would probably help on other POI types as well.

The idea would be to allow users to focus on only one type of POI, using some kind of filters / settings.

This would allow for much more efficient mapping in a "completionist" approach when working on making sure that some areas aren't missing even 1 POI of a specific type (could be used for bicycle parking, benchs, fire hoses, shops... 

I'm a believer that some people who are not OSM-informed might be highly motivated to work on one type of data in the map and organize their communities to micro map whole areas for their special interest, provided they have the tools to do so.

**Comment by Zverik** [comment]
Nice idea, and would probably be doable after #119. Closing this in favor of that.

In general, I'd say the editor is for mapping things directly around you, so other object types should not overwhelm a mapper if they want to focus on a single one.
